README.md

```diff
@@ -72,6 +72,7 @@ Commands:
   commit-queue  Submit changes from the base repository and any enabled modules to the...
   disable       Disable the specified module in the current repo.
   enable        Enable the specified module in the current repo.
+  git-commit    Perform git commit --all --message operations to all the modified modules.
```

Comment:
Was this generated from click? It reads really awkwardly. I don't think we should include "--all --message" in the summary, just in the detailed information.